### ISSUE TITLE
Move the container when an even happen

### ISSUE THREAD

**Issue by xescugc** [mcarpenter622/ebitenui#183]
On the same fashion as https://github.com/ebitenui/ebitenui/issues/160.

I have a button based on https://github.com/ebitenui/ebitenui/blob/master/_examples/widget_demos/imagebutton/main.go that has more things than an image but the same concept (is this https://github.com/ebitenui/ebitenui/issues/168).

So I was wondering in which way I could make all that content also **move down** when pressed.

I've tried something like:

```
				widget.WidgetOpts.MouseButtonPressedHandler(func(args *widget.WidgetMouseButtonPressedEventArgs) {
					fmt.Println("Pressed")
					args.Widget.Rect = stdimage.Rect(args.Widget.Rect.Min.X, args.Widget.Rect.Min.Y+2, args.Widget.Rect.Max.X, args.Widget.Rect.Max.Y+2)
				}),

```

Using `SetLocation` also does nothing.

so basically I would move the container pressed down 2 pixels but it's not moving anything haha.

**Comment by mcarpenter622** [comment]
What I am seeing here is we have a need for a "PressedInset" field for button that will affect all the content within it. Where it will apply that inset to the content inside the button (text or whatever) whenever the button is pressed. Does that sound correct?

**Comment by xescugc** [comment]
It makes sense, but in more general context the idea would be basically to be able to move a container dynamically, because at the end If I could change the X,Y that would also work
